root/static/scripts/release-editor/fields.js

```diff
@@ -130,15 +130,18 @@ class Track {
 
         // Convert stuff like 111 into 1:11
 
-        if (/^\d+$/.test(length) && (5 - lengthLength) <= 1) {
-            var minutes, seconds, hours;
+        if (/^\d+$/.test(length) && (6 - lengthLength) <= 1) {
+            var minutes, seconds, hours = null;
 
 
             if (lengthLength === 3) minutes = length[0];
             if (lengthLength === 4) minutes = length.slice(0, 2);
             if (lengthLength === 5) minutes = length.slice(1,3); {
                 hours = length.slice(0);
             }
+            if (lengthLength === 6) minutes = length.slice(2,4);{
```

Comment:
That is exactly the same issue than in [previous comment about split block of code](#r234411335).

Reply:
"minutes and hours computations are not in the same block of code because the opening curly bracket { is in the middle of it rather than in front of it. " I'm sorry I don't understand this statement, so to fix it I'm assuming I should move up the front bracket into of (lengthLength === 5), and chunk minutes = length.slice(1,3); into the {} brackets?

Reply:
1. Not exactly, let me try to clarify with simplified statements.

   a) What **you originally wrote** is in the form:
   ```javascript
   if (condition) statement1; {
     statement2;
   }
   ```
   which is effectively the same (with usual coding style) as:
   ```javascript
   if (condition) {
     statement1;
   }
   statement2;
   ```
   that is `statement2` will be executed even if `condition` resolves into `false`.

   b) What **I suggested** is to open curly bracket in front of `statement1` as follows:
   ```javascript
   if (condition) {statement1;
     statement2;
   }
   ```
   which is effectively the same (with usual coding style) as:
   ```javascript
   if (condition) {
     statement1;
     statement2;
   }
   ```
   Here `statement2` will be executed if and only if `condition` resolves into `true`.

2. Replacing simplified statements returns the following **actual suggestion**:
```suggestion
            if (lengthLength === 6) {
              minutes = length.slice(2,4);
```